scripts/github_release.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env zsh
```

Comment:
Can you provide a source to this file to how this file was developed?

Reply:
Not clear to me how this file is used, how is the tag chosen by the user?

Reply:
You'd choose tag, e.g., `v.4.0.2` and run `./scripts/github_release v.4.0.2`.

Reply:
Great, can you add that running instructions `./scripts/github_release v.4.0.2` including all the build step, as a subsection in https://quantus.readthedocs.io/en/latest/docs_dev/CONTRIBUTING.html?

Reply:
tbh, I am not a big fan of this script either. Let's try to catch up in the following days, and talk about how would you want to organize the release process.